.github/workflows/test-release.yml

```diff
@@ -0,0 +1,76 @@
+name: GraalVM Native Image builds
```

Comment:
Will we rename the file to just `release.yml`?
Also the name should be something like `Build and deploy a release including native-image`